tests/requirements.txt

```diff
@@ -1,8 +1,8 @@
 #
-# This file is autogenerated by pip-compile with Python 3.10
+# This file is autogenerated by pip-compile with Python 3.11
```

Comment:
This has the same problem as in https://github.com/ansible/ansible-documentation/pull/322.

Reply:
Thanks for catching that @gotmax23 should be fixed now